Css/allModals.css

```diff
@@ -0,0 +1,28 @@
+#uploadImageDiv {
+    margin-top: 5px;
+    margin-bottom: 10px;
+}
+
+#uploadImagelabel {
+    margin-top: 10px;
+}
+
+.modal-header {
+	background: #396060;
+}
+
+.modal-title-custom {
+	margin-left: 125px;
+	margin-right: auto;
+	color: white;
+}
+
+.projectModalSelection1 {
```

Comment:
Modal Selection 1? Not sure what you mean here, name it specifically to what modal you are referring to